### PR TITLE
Make duels dust filter options multiples of cards cost

### PR DESCRIPTION
**Pull request by piedpiper358**
**Before:**

**After:**

### CODE REVIEW

**Comment by sebastientromp** [comment]
I'll check with Duels players to see what they prefer

File: core/src/js/models/duels/duels-types.ts [hunk 1]
```diff
@@ -1,6 +1 @@
-export type DuelsTopDecksDustFilterType = 'all' | '1000' | '500' | '200' | '100' | '40' | '0';
-export type DuelsUnlocksFilterType =
```
Comment (sebastientromp) [inline core/src/js/models/duels/duels-types.ts:2]: Why did you remove that type declaration? I assume it's a mistake?

Comment (sebastientromp) [inline core/src/js/models/duels/duels-types.ts:2]: Why did you remove that type declaration? I assume it's a mistake?

Comment (piedpiper358) [reply core/src/js/models/duels/duels-types.ts:2]: Now all the treasures are open by default and you removed corresponding filter from app and I thought that we can also remove this declaration. This type is currently not used anywhere in the app.
